bucket/tabula.json

```
@@ -0,0 +1,31 @@
+{
+    "version": "1.2.1",
+    "description": "A tool for liberating data tables trapped inside PDF files.",
+    "homepage": "https://tabula.technology/",
+    "license": "MIT",
+    "url": "https://github.com/tabulapdf/tabula/releases/download/v1.2.1/tabula-win-1.2.1.zip",
+    "hash": "a5bd9afd969c7488f91ea3a7a09d56a1b53fe86ee231b81d205a8c1a9d459a05",
+    "extract_dir": "tabula",
+    "bin": "tabula.exe",
+    "shortcuts": [
+        [
+            "tabula.exe",
+            "tabula"
+        ]
+    ],
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "checkver": {
+        "github": "https://github.com/tabulapdf/tabula"
+    },
+    "autoupdate": {
+        "url": "https://github.com/tabulapdf/tabula/releases/download/v$version/tabula-win-$version.zip",
+        "hash": {
+            "url": "https://github.com/tabulapdf/tabula/releases/latest"
```

Never use latest for hash checks. This will get wrong hash shen installed as `tabula@version`

```suggestion
            "url": "https://github.com/tabulapdf/tabula/releases/tag/v$version"
```